nbs/02_maker.ipynb

```diff
@@ -248,6 +248,7 @@
     "#|export\n",
```

Comment:
<p>When is this needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/928/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>If you use <code>@fastcore.basics.patch</code> (or whatever the right import is) instead of importing directly using <code>from fastcore.basics import *</code> and then using <code>@patch</code> </p>